external-crates/move/crates/move-compiler/src/expansion/ast.rs

```diff
@@ -637,6 +637,14 @@ impl UseFuns {
 }
 
 impl Address {
+    pub fn new(name: Name, loc: Loc, address: NumericalAddress, name_conflict: bool) -> Self {
+        Self::Numerical {
+            name: Some(name),
+            value: sp(loc, address),
+            name_conflict,
+        }
+    }
```

Comment:
This feels rather strange to me since it is not clear which variant it will enter. Whats the motivation for this function?

Reply:
This struct's fields are private and I need to construct an instance in `move-analyzer`

Reply:
What issue are you having? It is a `pub enum` so I'm confused

Reply:
The issue is that I am a moron :-( Fixed!